cli_tools/google-osconfig-agent/patch/system_linux.go

```diff
@@ -0,0 +1,44 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package patch
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/logger"
+)
+
+// disableAutoUpdates disables system auto updates.
+func disableAutoUpdates() {
+	// yum-cron on el systems
+	if _, err := os.Stat("/usr/sbin/yum-cron"); err == nil {
```

Comment:
Could we check to see if it's on before turning it off? Then we can log that we've turned it off.